src/github.com/getlantern/lantern/lantern.go

```diff
@@ -124,15 +124,26 @@ func run(configDir string) {
 		make(map[string]interface{}),                  // no special configuration flags
 		func(cfg *config.Config) bool { return true }, // beforeStart()
 		func(cfg *config.Config) {},                   // afterStart()
-		func(cfg *config.Config) {},                   // onConfigUpdate
+		func(cfg *config.Config) {},                   // onConfigChange()
 		&userConfig{},
 		func(err error) {}, // onError
 	)
 }
 
+// CheckForUpdates checks to see if a new version of Lantern is available
+func CheckForUpdates(shouldProxy bool) (string, error) {
+	return autoupdate.CheckMobileUpdate(shouldProxy, config.DefaultUpdateServerURL,
```

Comment:
Change this from ``config.DefaultUpdateServerURL`` to the autoupdate staging server (``https://update-stage.getlantern.org/update``) during testing otherwise you will get an error about Android not being supported